homeassistant/components/sfr_box/strings.json

```diff
@@ -13,5 +13,33 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "line_status": {
+        "state": {
+          "no defect": "No Defect",
+          "of frame": "Of Frame",
+          "loss of signal": "Loss Of Signal",
+          "loss of power": "Loss Of Power",
+          "loss of signal quality": "Loss Of Signal Quality",
+          "unknown": "Unknown"
```

Comment:
This also looks odd; going to put this on the agenda to discuss. I should have feedback for you upcoming Thursday.

Reply:
I moved all other changes to separate PR, and rebased this one accordingly.

Reply:
Ok, so the relevant details can be found here: #85221 :)